Projects/DbCLI/SearchDb.py

```diff
@@ -421,6 +422,8 @@ def poolFromCurs(curs, similarityMethod):
     if not options.silent:
       logger.info('The search took %.1f seconds' % (t2 - t1))
 
+# Please check %.1f
+
```

Comment:
You can always try these things out in python:
```
python
Python 3.6.2 |Anaconda custom (x86_64)| (default, Jul 20 2017, 13:14:59) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 'The search took %.1f seconds' % 3,1415
('The search took 3.0 seconds', 1415)
>>> 
```
it works

Reply:
Ouh, never thought of putting it there. Thanks!